src/main/scala/io/github/davidgregory084/ScalacOptions.scala

```diff
@@ -39,6 +39,14 @@ trait ScalacOptions {
   val feature =
     new ScalacOption(List("-feature"))
 
+  /** Compile for a specific version of the Java platform. Supported targets: 8, 9, ..., 17, 18.
+    *
+    * The release flag is supported only on JDK 9 and above, since it relies on the functionality
+    * provided in [[http://openjdk.java.net/jeps/247 JEP-247: Compile for Older Platform Versions]].
+    */
+  def release(version: String) =
+    new ScalacOption(List("-release", version), version => version > V2_12_5)
```

Comment:
Note this flag has been renamed to `-java-output-version` in 3.1.2 as of https://github.com/lampepfl/dotty/pull/14606.

Reply:
Ah cool, I will spend some time next week to understand everything that has happened there. It looks like they have left the old names in as aliases luckily.